dimod/decorators.py

```diff
@@ -34,7 +40,13 @@
 
 
 def bqm_index_labels(f):
-    """todo
+    """Decorator to convert a bqm to index-labels and relabel the sample set
+    output.
+
+    Designed to be applied to :meth:`.Sampler.sample`. Expects the wrapped
```

Comment:
Applied to :meth:`.Sampler.sample`.

Reply:
No, that is not correct. It is designed to be applied to that method, but it could be used elsewhere